src/utils/sortObjectKeys.ts

```diff
@@ -1,20 +1,26 @@
-import { Settings, SortOrder } from "../types/settings";
+import { Settings, SortOrder } from "../types/settings.d";
+import { GenericObject } from "../types/shared";
 
 const sortHandlers = {
-  [SortOrder.ASC]: (object: Record<string, any>) =>
+  [SortOrder.ASC]: (object: GenericObject) =>
     Object.fromEntries(
       Object.entries(object).sort(([a], [b]) => a.localeCompare(b))
     ),
 
-  [SortOrder.DESC]: (object: Record<string, any>) =>
+  [SortOrder.DESC]: (object: GenericObject) =>
     Object.fromEntries(
       Object.entries(object).sort(([a], [b]) => b.localeCompare(a))
     ),
 };
 
 function sortObjectKeys(obj: any, sort: SortOrder = SortOrder.NONE): any {
-  if (sort === SortOrder.NONE || typeof obj !== "object" || obj === null)
+  if (!Object.values(SortOrder).includes(sort)) {
+    throw new Error("Invalid sort value");
+  }
```

Comment:
const sortHandler = sortHandlers[sort];

if (!sortHandler) {
   throw new Error(`The sort option you've provided is not supported, please try one of the next: Object.values(SortOrder).join(', ')`);
}

Reply:
this way you can remove the ternary operator below:

![image](https://github.com/user-attachments/assets/8c623e04-01fd-4d12-8152-1830c2f13e82)

Since you'll now that it exists a sort handler and can avoid that conditional